packages/general-types/src/constants/slot.ts

```diff
@@ -398,6 +537,27 @@ export const SlotTypes: ObjectKeys = {
     },
     NUMBER('Numero'),
     PERCENTAGE('Percentuale'),
+    ORDINAL('Numero ordinale'),
+    URL('URL'),
+    KEY_PHRASE('Frase chiave'),
+    {
+      name: SlotType.CURRENCY,
+      label: 'Valuta',
+      values: ['5 dollari', '9.10 dollari canadesi', '6 sterline', '$4.1', 'quattro pence', '24 pesos', '84 £', '£99.12'],
+    },
+    {
+      name: SlotType.DIMENSION,
+      label: 'Dimensione',
+      values: ['10 1/2 miglia', '10,5 chilometri', '0,3 mm', '5 piedi', '6 iarde', '8m'],
+    },
+    {
+      name: SlotType.TEMPERATURE,
+      label: 'Temperatura',
+      values: ['30 gradi', '215 kelvin', '56 f', '33°f', '22 gradi Celsius', '102 gradi fanheit'],
+    },
   ],
-  [Language.KO]: [EMAIL, PHONENUMBER, PERCENTAGE('백분율')],
+  [Language.KO]: [EMAIL, PHONENUMBER, URL('URL'), KEY_PHRASE('핵심 문구')],
+  // Mexico Spanish has much less built in entities than Spain Spanish.
+  // There is a case in getSlotTypes in realtime-sdk for this
+  [Locale.ES_MX]: [EMAIL, NUMBER('Número'), URL('URL'), KEY_PHRASE('Frase clave')],
```

Comment:
Mexico spanish has a lot less entities than spain spanish so need a seperate case for it. Where this is used, they take the substring to remove the locale part and keep the language. See here: https://github.com/voiceflow/creator-app/blob/master/packages/realtime-sdk/src/utils/slot.ts#:~:text=default%3A-,builtInSlots,-%3D%20generalSlotTypesByLanguage(locales%5B0. A PR in Creator-App is made for this as well